driver/parser/check.go

```diff
@@ -259,3 +271,29 @@ func checkTimeInterval(start, end *float32, duration float32) error {
 	}
 	return errors.Join(errs...)
 }
+
+func checkTimeNodeAlive(start *float32, node *Node, duration float32) error {
```

Comment:
What does this function intend to check? It is used by both Event.Import and Event.Export to check its start time. It looks the me that the start time must be between Node.Start < EventStart < Node.End. Similar to genesis, Export event can only be done after a node is stopped.

Reply:
You're right; the function does check `Node.Start < EventStart < Node.End`. 
Will create an inverted function and use that instead.

Reply:
We do have plan to remove `Start` in a later PR. The fix may not come immediately. I would prefer if this change is a standalone and works as intended. Can you either remove this function of fix its logic such that Import.Start = Node.Start and Export.Start = Node.End? Everything else looks good.

Reply:
`checkTimeNodeAlive` is removed.